Templates/Swift/Includes/Enum.stencil

```diff
@@ -1,14 +1,8 @@
 {% if description %}
 /** {{ description }} */
 {% endif %}
-public enum {{ enumName }}: {{ type }}, Codable {
+public enum {{ enumName }}: {{ type }}, Codable, Equatable, CaseIterable {
```

Comment:
I think you can conditionally use the template for swift 4.2 Vs 4.1

```swift
#if swift(>=4.2)
// swift 4.2 goes here
#else 
// swift 4.1 goes here 
#endif
````

Reply:
Good idea I will do it.

Reply:
I have done it but I still have to fix the fixture...